SConstruct

```diff
@@ -367,6 +387,35 @@ for d in os.listdir('.'):
         if os.path.exists(script):
             SConscript(script, variant_dir=join(GetOption('build-dir'), d), duplicate=0)
 
+if enscons_active:
+    Import('python_shlib')
+    Import('python_source')
+    Import('soarlib')
+    py_lib_namespace = env['PACKAGE_METADATA']['name'].replace("-", "_")
+
+    sources = []
+
+    if sys.platform == 'darwin' or os.name == 'nt':
+        # Add soar's library to the wheel directory
+        sources += [
+            env.Install(py_lib_namespace, soarlib)
+        ]
+
+    if sys.platform == 'darwin':
+        # For MacOS, also add to the out/ directory,
+        # so the linker and delocator pick up on it properly.
+        sources += [
+            env.Install(env['OUT_DIR'], soarlib)
+        ]
+
+    sources += [
+        env.Install(py_lib_namespace, python_shlib),
+        env.InstallAs(py_lib_namespace + "/__init__.py", python_source)
+    ]
+
+    whl = env.Whl("platlib", sources, root="")
+    env.WhlFile(source=whl)
```

Comment:
Specifying these lines adds a bunch of aliases to scons' build targets that cibuildwheel requires. It basically makes cibuildwheel work at all.

Reply:
Could you add a quick comment that says something to that effect in the source?